server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -222,27 +223,22 @@ while read size result ;do
     mkdir -p $RESULTS/$hostname/$prefix
     status=$?
     if [[ $status -ne 0 ]] ;then
-        echo "$TS: prefix failure - code $status:         mkdir -p $RESULTS/$hostname/$prefix" >&4
+        echo "$TS: code $status: mkdir -p $RESULTS/$hostname/$prefix" >&4
     else
```

Comment:
Here's where you have to check whether the unpack path is in the incoming directory (which will be the case for most servers, in particular satellites), or whether it's different (for the production server). In the former case, you *don't want to make the link* from incoming to unpack, since there will already be a directory in incoming (= unpack).